src/routes/safe/components/Balances/SendModal/screens/PayoutBounty/index.jsx

```diff
@@ -146,7 +146,7 @@ const PayoutBounty = ({ initialValues, onClose, onNext }: Props) => {
                       placeholder="Link to bounty ticket"
                       text="Link to bounty ticket"
                       type="text"
-                      validate={required}
+                      validate={composeValidators(required, minMaxLength(1, 32))}
```

Comment:
The length of the string will almost always exceed 32 characters, maybe check the length after removing the `https://github.com/leapdao/` bit.

Reply:
very good point 🤦‍♂️